minisom.py

```diff
@@ -435,7 +435,7 @@ def train_batch(self, data, num_iteration, verbose=False):
         """
         self.train(data, num_iteration, random_order=False, verbose=verbose)
 
-    def distance_map(self, neighbour_average=False):
+    def distance_map(self, scaling='sum'):
```

Comment:
changes look great. Can you fix the documentation of the method and a check to validate scaling?

Here's an example of how parameters are checked in other parts of the library https://github.com/JustGlowing/minisom/blob/master/minisom.py#L169

After this I'll run the tests myself and, if they pass, I'll merge the branch.